src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java

```diff
@@ -1796,6 +1878,15 @@ private JComboBox<MovementType> getJComboCharge() {
 			}
 			chargeCombo.addActionListener(actionEvent -> {
 				chargeType = (MovementType) chargeCombo.getSelectedItem();
+				if (inventory != null && chargeType != null && inventory.getChargeType() != null && !inventory.getChargeType().equals(chargeType.getCode())) {
+	    			 disableValidateButton();
+	    		} else {
+	    			if (inventory == null || chargeType == null) {
+	    				 disableValidateButton();
+	    			} else {
+	    				validateButton.setEnabled(true);	
+	    			}
+	    		}
```

Comment:
indentation doesn't look good (also in the other action events below).
anyway what is the aim of disabling the validate button in so many places and conditions?